atc/wrappa/concurrent_request_policy.go

```diff
@@ -0,0 +1,145 @@
+package wrappa
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/concourse/concourse/atc"
+)
+
+type ConcurrentRequestLimitFlag struct {
+	Action string
+	Limit  int
+}
+
+func (crl *ConcurrentRequestLimitFlag) UnmarshalFlag(value string) error {
+	variable, expression, err := parseAssignment(value)
+	if err != nil {
+		return err
+	}
+	limit, err := strconv.Atoi(expression)
+	if err != nil || limit < 0 {
```

Comment:
If we use https://golang.org/pkg/strconv/#ParseUint, I guess we don't need to check against negative numbers...